src/lib/BarStore.js

```diff
@@ -191,7 +193,10 @@ export class Provider extends Component {
     this.store = props.store
   }
 
-  render ({children}) {
+  render () {
+    const { children } = this.props
```

Comment:
why don't you keep it in the arguments ?

Reply:
That's a preact way, not a React way so it would break all tests that use React.

Reply:
Ah yes I see.